charts/opentelemetry-collector/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
   - name: pjanotti
   - name: tigrannajaryan
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: 0.22.0
+appVersion: v0.36.0
```

Comment:
The default configuration has jaeger and zipkin receivers which are not part of the core image anymore. So we should switch default image to contrib as well